validator/specs/lib/validateTilesetSpec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var validateTileset = require('../../lib/validateTileset');
+var clone = require('clone');
```

Comment:
You need to `npm install clone --save`. In `package.json` remove the `^` that is added since we prefer to lock our library versions.